tests/foreman/ui/test_classparameters.py

```diff
@@ -467,3 +473,822 @@ def test_negative_validate_matcher_and_default_value(self):
 
         @status: Manual
         """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_negative_validate_matcher_non_existing_attribute(self):
+        """Error while creating matcher for Non Existing Attribute.
+
+        @feature: Class Parameters - Validation
+
+        @steps:
+
```

Comment:
@sghai It is good to have this blank link otherwise the documentation prints everything in one line.